### PR TITLE
[Versioning] Fix CHANGELOG for 3.3.0

### PR DESCRIPTION
**Pull request by gmarciani**
### Description of changes
Fix CHANGELOG for 3.3.0

### Tests
Not needed.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

Signed-off-by: Giacomo Marciani <mgiacomo@amazon.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,33 +9,35 @@ This file is used to list changes made in each version of the AWS ParallelCluste
...
-  - Libfabric-aws: `libfabric-aws-1.16.0~amzn4.0-1`
-  - Rdma-core: `rdma-core-41.0-2`
-  - Open MPI: `openmpi40-aws-4.1.4-2`
-- Upgrade NICE DCV to version `2022.1-13300`.
-  - server: `2022.1.13300-1`
-  - xdcv: `2022.1.433-1`
-  - gl: `2022.1.973-1`
-  - web_viewer: `2022.1.13300-1`
+- Add support for compute resources with multiple instance types.
+- Add support for mounting and unmounting shared storage at cluster update.
```
Comment (enrico-usai) [inline CHANGELOG.md]: - Add support for adding or removing shared storages at cluster update.

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,33 +9,35 @@ This file is used to list changes made in each version of the AWS ParallelCluste
...
-  - Efa-profile: `efa-profile-1.5-1`
-  - Libfabric-aws: `libfabric-aws-1.16.0~amzn4.0-1`
-  - Rdma-core: `rdma-core-41.0-2`
-  - Open MPI: `openmpi40-aws-4.1.4-2`
-- Upgrade NICE DCV to version `2022.1-13300`.
-  - server: `2022.1.13300-1`
-  - xdcv: `2022.1.433-1`
-  - gl: `2022.1.973-1`
-  - web_viewer: `2022.1.13300-1`
+- Add support for compute resources with multiple instance types.
```
Comment (enrico-usai) [inline CHANGELOG.md]: - Add possibility to specify multiple instance types in a compute resource.

(or: in the same compute resource)

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,33 +9,35 @@ This file is used to list changes made in each version of the AWS ParallelCluste
...
+- Add support for mounting and unmounting shared storage at cluster update.
 - Configure NFS threads to be `min(256, max(8, num_cores * 4))` to ensure better stability and performance.
+- Move NFS installation at build time to reduce configuration time.
```
Comment (enrico-usai) [inline CHANGELOG.md:15]: I'd move into CHANGES, as:
- Install NFS at image build time.

File: CHANGELOG.md (hunk 1 above)
Comment (gmarciani) [reply CHANGELOG.md]: I agree with this more high level rephrasing.

File: CHANGELOG.md (hunk 2 above)
Comment (gmarciani) [reply CHANGELOG.md]: Agree

File: CHANGELOG.md (hunk 3 above)
Comment (gmarciani) [reply CHANGELOG.md:15]: That change is providing a performance improvement. I think it's worth it to mention in enhancement.
Why do you think it's not?

**Review by enrico-usai — APPROVED** [review]

**Pull request merged**